### PR TITLE
Fixing various incorrect links

### PR DESCRIPTION
**Pull request by TuanaCelik**
Some of these are just wrong as is. No point in adding them to redirects. So I've manually fixed them.

### CODE REVIEW

File: docs/v1.4.0/overview/faq.mdx [hunk 1]
```diff
@@ -43,16 +43,16 @@ and also our [blogs](https://medium.com/deepset-ai)
...
 have to set certain parameters or load models pretrained for your language in order to get good performance out of Haystack.
-See our [languages page](https://haystack.deepset.ai/guides/v1.4.0/languages) for more details.
+See our [languages page](/guides/v1.4.0/languages) for more details.
 
 <div style={{ marginBottom: "3rem" }} />
 
 ## How can I add metadata to my documents so that I can apply filters?
 
-When providing your documents in the input format (see [here](https://haystack.deepset.aihttps://haystack.deepset.ai/usage/v1.4.0/document-store#input-format))
+When providing your documents in the input format (see [here](/usage/v1.4.0/document-store#input-format))
```
Comment (masci) [inline docs/v1.4.0/overview/faq.mdx]: This one gives me 404 in my local env

File: docs/v1.4.0/overview/faq.mdx [hunk 2]
```diff
@@ -94,5 +94,5 @@ Note that this also applies at evaluation where labels are written into their ow
 In short, the FARMReader uses a QA pipeline implementation that comes from our own
 [FARM framework](https://github.com/deepset-ai/FARM) that we can more easily update and also optimize for performance.
 By contrast, the TransformersReader uses a QA pipeline implementation that comes from HuggingFace's [Transformers](https://github.com/huggingface/transformers).
-See [this section](https://haystack.deepset.ai/pipeline_nodes/v1.4.0/reader#deeper-dive-farm-vs-transformers)
+See [this section](pipeline_nodes/v1.4.0/reader#deeper-dive-farm-vs-transformers)
```
Comment (masci) [inline docs/v1.4.0/overview/faq.mdx]: This gives me 404 locally

**Review by masci — CHANGES_REQUESTED** [review]
I've found a couple of 404s that show up both locally and on live preview

**Comment by TuanaCelik** [comment]
Thanks @masci  - A missing forward slash and usage -> components. Fixed these

**Review by masci — APPROVED** [review]
LGTM! 🚀

**Pull request merged**